example/namedtracer/foo/foo.go

```diff
@@ -29,19 +29,14 @@ var (
 // SubOperation is an example to demonstrate the use of named tracer.
 // It creates a named tracer with its package path.
 func SubOperation(ctx context.Context) error {
-
 	// Using global provider. Alternative is to have application provide a getter
 	// for its component to get the instance of the provider.
 	tr := global.Tracer("example/namedtracer/foo")
-	return tr.WithSpan(
-		ctx,
-		"Sub operation...",
-		func(ctx context.Context) error {
-			trace.SpanFromContext(ctx).SetAttributes(lemonsKey.String("five"))
 
-			trace.SpanFromContext(ctx).AddEvent(ctx, "Sub span event")
+	var span trace.Span
+	ctx, span = tr.Start(ctx, "Sub operation...")
```

Comment:
It doesn't look like this span ends.  Is that intentional?

Reply:
Ah, it was not intentional. Good catch.